src/conda/env_python3_base.yml

```diff
@@ -14,3 +14,6 @@ dependencies:
 - xarray=0.16
 - matplotlib=3.3
 - cartopy=0.18
+- numba = 0.51.2
+- networkx = 2.5
+- jupyterlab
```

Comment:
`numba` may force a version change in `numpy` in my experience.  How are we going to evaluate all of the PODs before and after this change?

Reply:
By specifying numpy version?

Reply:
Sure. Or at least a minimum version, @yihungkuo 

I am thinking we need to run the package first, save the output, run again with these updates, and then compare the output.  @yihungkuo, @wrongkindofdoctor, or @tsjackson-noaa - can you think of an objective way of comparing two sets of MDTF output?

Reply:
@jkrasting, @wrongkindofdoctor There may be an automated approach to compare binary or intermediate netCDF files. But I suspect that some kind of visual inspection for figures would be inevitable. During phase 1 (& 2), there were times when major matplotlib updates made the figures from the UCLA POD unreadable--the figures were generated successfully, but the aspect ratio, font size, etc., all went wrong.